libs/csrf/csrfprotector.php

```diff
@@ -354,7 +354,7 @@ public static function refreshToken()
 		public static function generateAuthToken()
 		{
 			// todo - make this a member method / configurable
-			$randLength = 32;
+			$randLength = 64;
```

Comment:
Because you have defined that the maximum length for the token is 128 characters. Keeping it as 32 would've given only 64 characters.
```php
<?php
var_dump(bin2hex(random_bytes(32)));
var_dump(bin2hex(random_bytes(64)));
```
output:
```
string(64) "a0b6a05528a1c3e84f49166929e4a9e38dd4428fcec1532f162f40b292cf4a9f"
string(128) "89fd768dc37c56dfd92a1b5ce421fe7633dfcc26bad109828222dcb217ba5b2bd0aa84bf131febb33c43421182c2361f375e8166a272f8c88e37f6e2fc71ffcf"
```